src/ensembl/src/shared/components/layout/components/sidebar-modal/SidebarModal.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactNode } from 'react';
+import classNamesMerger from 'classnames';
+
+import CloseButton from 'src/shared/components/close-button/CloseButton';
+
+import styles from './SidebarModal.scss';
+
+export type SidebarModalProps = {
+  title: ReactNode;
```

Comment:
Shouldn't title be a string here?

Reply:
That's done

Reply:
![image](https://user-images.githubusercontent.com/6834224/142464095-ebe9c2bd-4439-4b9f-8d5f-8628de8f8dc2.png)

Reply:
Fixed it 👍